projects/plugins/boost/app/modules/optimizations/lcp/class-lcp.php

```diff
@@ -377,6 +378,59 @@ private function optimize_image( $tag ) {
 			$tag = preg_replace( '/<img\s/i', '<img loading="eager" ', $tag );
 		}
 
+		if ( ! preg_match( '/src\s*=\s*["\']([^"\']+)["\']/i', $tag, $matches ) ) {
+			return $tag;
+		}
+
+		$image_url = $matches[1];
+
+		// Update the src attribute with the CDN URL
+		$tag = str_replace(
```

Comment:
This seems to work for me, but I was wondering if we should add a conditional check to ensure that the image CDN module is active before we start using the CDN on a site's images?